deploy/roles/default_role.yaml

```diff
@@ -30,6 +30,19 @@ rules:
       - batch
     resources:
       - jobs
+  - verbs:
```

Comment:
Need to be checked in PSI and app-sre
I'm not sure that you have the permissions there.

Reply:
interesting. Not sure it can work without having them. Who can i talk to?

Reply:
on the other hand it is much less permissions then to start job

Reply:
still need to verify
@romfreiman